src/components/CreateMatchExperienceModal/index.tsx

```diff
@@ -41,16 +41,20 @@ const CreateMatchExperienceModal = ({ isOpen, onClose }: CreateMatchExperienceMo
     const [imageUrl, setImageUrl] = useState<string>('');
     const [selectedCountry, setSelectedCountry] = useState<string>('');
     const [selectedLeague, setSelectedLeague] = useState<number>();
+    const [selectedDate, setSelectedDate] = useState<Date | null>(null);
```

Comment:
```suggestion
const {
        control,
        handleSubmit,
        setValue,
        reset,
        getValues,
        trigger,
        formState: { errors },
    } = useForm<CreateMatchExperienceModalValues>({
        resolver: yupResolver(MatchExperienceSchema),
        defaultValues: {
            title: '',
            description: '',
            picture: '',
            country: '',
            stadium: '',
            league: '',
            homeTeam: '',
            awayTeam: '',
            matchDate: null,
        },
    });
```

dont make `useStates` just use form as the state manager. unless there is a good reason for that

Reply:
in any case the `defaultValues` will be later refactored for edit mode of form that I will do

Reply:
Some types are saved as strings but are used as numbers (like the league, for example), and for that reason, I need to use `useState`